### ISSUE TITLE
Setting up TV , getting error "bash: python3.8: command not found"

### ISSUE THREAD

**Issue by ColinKlayer** [MrAsterisco/homebridge-hisense-tv#17]
**Describe Your Problem:**

I am attempting to follow the "Setting up the TV" directions pasted below but am getting an error response. 

https://github.com/MrAsterisco/homebridge-hisense-tv
> Setting up the TV
> First, you need to get the name of the network interface that your Homebridge machine will use to connect to the TV. On Linux, you can get a list of all the network interfaces on your machine by running:
> 
> ip a
> The name of a network interface usually looks similar to this: ens33.
> 
> For this plugin to work correctly, you need to configure your TV to use a static DHCP (or configure a static reservation on your router). You also need to find your TV's MAC Address: this is usually displayed under Settings > Network Information, but it might vary based on your model.
> 
> To connect to your TV, you need to pair the machine where you're running Homebridge with your TV. This is done in the command line, by manually running the bundled hisensetv.py script. To do this, find the node_modules folder in your system (on Linux, it is located in /usr/local/lib/node_modules) and move to homebridge-hisense-tv/bin, then run:
> 
> python3.8 hisensetv.py <TV_IP_ADDRESS> --authorize --ifname <NETWORK_INTERFACE_NAME>
> Your TV, if compatible, will display a PIN code: insert it in the command line and confirm. Your device is now paired with your TV and they can communicate when the TV is on. Repeat this step for all the TVs you want to use via HomeKit.
> 
> If the command times-out after a while, make sure your TV is connected to the network and turned on. You can try to telnet <TV_IP_ADDRESS> 36669 to make sure your Homebridge instance can reach your TV. If telnet works but this command doesn't, try to run a different command, such as --get state (just replace authorize with this in the command above): if this command succeeds, it means your TV and your machine are already paired.

**From this location**
`homebridge:/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin`

**I run this command:**
`python3.8 hisensetv.py 192.168.X.XXX --authorize --ifname wlan01`

**Response:** 
`bash: python3.8: command not found`

**Logs:**

```
[8/21/2021, 1:57:29 PM] ---
[8/21/2021, 1:57:29 PM] Loaded plugin: homebridge-hisense-tv-remotenow@1.1.0
[8/21/2021, 1:57:29 PM] Registering platform 'homebridge-hisense-tv-remotenow.HiSenseTV'
[8/21/2021, 1:57:29 PM] ---
[8/21/2021, 1:57:39 PM] [HiSenseTV] Initializing HiSenseTV platform...
[8/21/2021, 1:57:40 PM] [HiSenseTV] Adding new accessory: Sly TV
Preparing Advertiser for 'Sly TV 40CE' using bonjour-hap backend!
Starting to advertise 'Sly TV 40CE' using bonjour-hap backend!
[8/21/2021, 1:57:41 PM] Sly TV 40CE is running on port 44705.
[8/21/2021, 1:57:41 PM] Please add [Sly TV 40CE] manually in Home app. Setup Code: 863-75-332
[8/21/2021, 1:57:51 PM] [HiSenseTV] An error occurred while fetching inputs: TypeError: Cannot read property 'join' of null
[8/21/2021, 1:58:01 PM] [HiSenseTV] An error occurred while fetching inputs: TypeError: Cannot read property 'join' of null
[8/21/2021, 1:58:21 PM] [HiSenseTV] An error occurred while fetching inputs: TypeError: Cannot read property 'join' of null
[8/21/2021, 1:58:31 PM] [HiSenseTV] An error occurred while fetching inputs: TypeError: Cannot read property 'join' of null
[8/21/2021, 1:58:41 PM] [HiSenseTV] An error occurred while fetching inputs: TypeError: Cannot read property 'join' of null
```

**Plugin Config:**

```json
{
    "ifname": "wlan01",
    "devices": [
        {
            "id": "Sly TV",
            "name": "Sly TV",
            "ipaddress": "192.168.X.XXX",
            "macaddress": "XX:XX:XX:XX:XX:XX",
            "sslmode": "disabled"
        }
    ],
    "platform": "HiSenseTV"
```

**Environment:**

* **Plugin Version**: homebridge-hisense-tv-remotenow v1.1.0
* **Homebridge Version**: homebridge -v1.3.4
* **Node.js Version**: node -v14.17.5
* **NPM Version**: npm -v6.14.14
* **Operating System**:  Raspbian GNU/Linux Buster (10)

**Comment by chinedu40** [comment]
Are you able to set your homebridge to debug mode and paste the logs it gives more detailed information

**Comment by MrAsterisco** [comment]
@ColinKlayer it seems you don't have `python3.8` installed or in your PATH.

I am not familiar with the Raspberry OS, but I did a quick Google search and found this: https://installvirtual.com/how-to-install-python-3-8-on-raspberry-pi-raspbian/

I'm closing this issue for now, as it's not related to the plugin itself.

In #15 , I suggested we update the documentation so that it includes all the steps in order to use this plugin on Raspbian. Feel free to drop your experience there as well. Unfortunately, I cannot do it myself, as I don't have a Raspberry machine.
